### PR TITLE
feat: add experiments details page and endpoint [DET-3003]

### PR DESCRIPTION
**Pull request by hamidzr**
## Commentary (optional)

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: webui/react/src/services/api.ts [hunk 1]
```diff
@@ -27,13 +28,21 @@ export const isLoginFailure = (e: any): boolean => {
   return e.response && e.response.status && e.response.status === 403;
 };
 
+/* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+export const isNotFound = (e: any): boolean => {
```
Comment (hkang1) [inline webui/react/src/services/api.ts:32]: question: for future use?

Comment (hamidzr) [reply webui/react/src/services/api.ts:32]: nah seems like I forgot to use it. https://github.com/determined-ai/determined/pull/795/files#diff-afa4dcb145f465ed696ff6ee10e2781bR29 we would check the error to see if it's actually a 404 or some other error.

Comment (hamidzr) [reply webui/react/src/services/api.ts:32]: follow up to that would be https://determinedai.atlassian.net/browse/DET-3500?filter=-2

**Comment by hamidzr** [comment]
Screen Capture_select-area_20200701143536
Screen Capture_select-area_20200701143434

File: webui/react/src/services/api.ts (hunk 1 above)
Comment (hamidzr) [reply webui/react/src/services/api.ts:32]: added that logic. (ss below)

**Pull request merged**